crates/ursa-proxy/src/core/handler.rs

```diff
@@ -199,6 +200,10 @@ pub async fn proxy_pass<C: Cache>(
                             }
                         }
                     }
+                    if skip_cache {
```

Comment:
Related to comment above
```suggestion
                    if max_size_cache_entry > 0 && bytes.len() > max_size_cache_entry {
```